src/prpy/rave.py

```diff
@@ -235,3 +235,229 @@ def disable_padding(body, enable=False):
     for link in body.GetLinks():
         if link.GetName().startswith('padding_'):
             link.Enable(enable)
+
+
+def add_box(env, name, x_len=0.1, y_len=0.1, z_len=0.1):
```

Comment:
It would be useful to have (optional), `(x, y, z)` arguments, just like we have in `resize_box`.